metadata-ingestion/tests/unit/test_kafka_source.py

```diff
@@ -376,7 +376,7 @@ def test_kafka_ignore_warnings_on_schema_type(
         schema_id="schema_id_2",
         schema=Schema(
             schema_str="{}",
-            schema_type="JSON",
+            schema_type="UNKNOWN_TYPE",
```

Comment:
This is to prevent schema parsing? Anything we'd want to do here to try out the schema parsing?

Reply:
Previously, we were asserting that if confluent schema registry sends us back JSON schemas, we would fail and we have a config flag to control how the connector behaves when we encounter a schema type we dont handle. Now since JSON schemas actually work, this test fails, so I had to change this to pass in a schema type that we don't handle.